src/Unleash/UnleashContext.cs

```diff
@@ -6,16 +6,8 @@ namespace Unleash
     /// <summary>
     /// A context which the feature request should be validated against. Usually scoped to a web request through an implementation of IUnleashContextProvider.
     /// </summary>
-    public class UnleashContext
+    public class UnleashContext : Yggdrasil.Context
```

Comment:
This made things easier to work with. But not sure if I'm a fan of introducing object model reliance on Yggdrasil.